test/gtest/uct/test_md.cc

```diff
@@ -94,9 +94,12 @@ void test_md::modify_config(const std::string& name, const std::string& value,
     }
 }
 
-bool test_md::check_caps(uint64_t flags)
+bool test_md::check_caps(uint64_t flags, ucs_memory_type_t mem_type)
```

Comment:
i think it should be in new function since we could have different kinds of memtype checks

check_reg_mem_type(ucs_memory_type_t mem_type)
{
return check_caps(REG)&& m_md_attr.cap.reg_mem_types & UCS_BIT(mem_type)
}